pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -703,55 +703,82 @@ func TestSyncEndpointsItemsPreexistingLabelsChange(t *testing.T) {
 
 func TestWaitsForAllInformersToBeSynced2(t *testing.T) {
 	var tests = []struct {
+		name                  string
 		podsSynced            func() bool
 		servicesSynced        func() bool
 		endpointsSynced       func() bool
 		shouldUpdateEndpoints bool
 	}{
-		{neverReady, alwaysReady, alwaysReady, false},
-		{alwaysReady, neverReady, alwaysReady, false},
-		{alwaysReady, alwaysReady, neverReady, false},
-		{alwaysReady, alwaysReady, alwaysReady, true},
+		{
+			"Should not update endpoints with podsSynced never ready, servicesSynced always ready, endpointsSynced always ready",
```

Comment:
thanks for the PR -- can you add the field names here, so it is easier for future updates to the tests:
```
{
  name: "Should not ... ",
  podsSynced: ...,
  ...
}
```